frontend/src/pages/DashboardPage/TimeButtons/TimeButtons.jsx

```diff
@@ -0,0 +1,36 @@
+// TimeButtons.jsx
```

Comment:
Is this component reusable?

Reply:
That's what I was thinking. If so, this is a really nice implementation so we can adjust the CSS as needed

Reply:
Yes, it's reusable